lib/types/models.ts

```diff
@@ -205,6 +206,15 @@ export interface PublicOrganization {
 	handle: string;
 }
 
+export interface PublicDevice {
+	id: string;
+	latitude: string;
+	longitude: string;
+	belongs_to__application: NavigationResource<Application>;
```

Comment:
This and `is_of__device_type` is not returned by the api yet!

Reply:
Removed `id`, however `is_of__device_type` is being returned if I run:

```
await window.sdk.pine.get({
    resource: 'public_device',
    options: {  $top: 1 }
});
```

I get: 
```
[{…}]
0:
belongs_to__application: {__id: 1635297, __deferred: {…}}
is_of__device_type: {__id: 145, __deferred: {…}}
latitude: ""
longitude: ""
was_recently_online: false
```

Reply:
It indeed does return them now :tada: